capstone-frontend/src/components/ClubGridItem.jsx

```diff
@@ -17,22 +17,22 @@ class ClubGridItem extends Component {
       id: this.props.club.id,
```

Comment:
I feel like these formatting changes should be in another PR, it doesn't look like you touched this file other than the GitPod formatting